verticapy/tests/base.py

```diff
@@ -208,3 +209,27 @@ def require_DB_version_at_least(self, min_version):
         pass
 
     # Common assertions
+
+
+base_test = VerticaPyTestBase()
+base_test.setUp()
+
+if not (
+    os.path.isfile(os.path.dirname(verticapy.__file__) + "/tests/verticaPy_test.conf")
+):
+    path = os.path.dirname(verticapy.__file__) + "/tests/verticaPy_test.conf"
+    confparser = ConfigParser()
+    confparser.optionxform = str
+    try:
+        confparser.read(path)
+    except:
+        pass
+    if confparser.has_section("vp_test_config"):
+        confparser.remove_section("vp_test_config")
+    confparser.add_section("vp_test_config")
+    for elem in base_test.test_config:
+        if elem != "log_level":
+            confparser.set("vp_test_config", elem, str(base_test.test_config[elem]))
+    f = open(path, "w+")
+    confparser.write(f)
+    f.close()
```

Comment:
Apparently, you don't clean a potentially created file, after the tests are done.
I think it would be nicer if you move the code under the `if not` into a function defined in conftest.py. Also, define another function over there that deletes the file. Then, you can call the create_file function before the `yield`, and the delete_file function after  the `yield` in the conftest.base() function.
Also, I suggest you name the file different from vp_test_config, so we won't over write it if it already exists.

Reply:
Done;

Reply:
When do you push your fixes to the PR?